### DISCUSSION TITLE
Will it work on android via termux

### DISCUSSION THREAD

**Discussion by tathastu871** [ArchiveBox/ArchiveBox#1149] [General]
How much functionl is archivebox on android platfim va termux app

**Comment by pirate** [comment]
I'm going to guess very little because the extractors need to work too (wget, chromium, yt-dlp, etc.). It's not just pure python, it relies on a lot of compiled binaries.

**Reply by tathastu871** [reply]
Termux has native wget yt-dlp and chromium. Also termuc has arm64 proot linux distro support

**Reply by pirate** [reply]
oh then maybe worth  shot, I don't know enough to say either way
